src/Domain/Model/BasePaymentMethod.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\PaymentContext\Domain\Model;
+
+abstract class BasePaymentMethod implements PaymentMethod {
```

Comment:
To be extra hip you could try using a trait :)

My experience with those is limited, but I don't see much differences with the base class (the big upsides and downsides being the same). Some minor ones in favor of a trait though:
* Trait can be dedicated to one thing, rather than be a bucket for "all things that some subclasses need"
* You can't accidentally use `BasePaymentMethod` as a type hint where you should be using `PaymentMethod`. If you look at a subclass it might not be clear there is an interface. Actually not sure if you can type hint against a trait... I suppose you can, but if the name is specific such confoosion is much less likely.